NEMO/models.py

```diff
@@ -1674,4 +1674,4 @@ def record_active_state(request, obj, form, field_name, is_initial_creation):
 		activity_entry.authorizer = request.user
 		activity_entry.action = getattr(obj, field_name)
 		activity_entry.content_object = obj
-		activity_entry.save()
+		activity_entry.save()
```

Comment:
I don't modify this file ! Maybe a typo or bug between Visual code and Github...